main/getting-started/alpha.md

```diff
@@ -0,0 +1,60 @@
+# Agoric Alpha	
+Release Version: Adriatic
+
+![Alpha Cover](./assets/alpha-cover.png)
+
+We’re building components to power the DeFi ecosystem. Come take a peek under the hood and help us scale our next-gen infrastructure. The Alpha Adriatic release provides you access to our toolset, API, and public chain.
```

Comment:
I'm not sure is "scale" is the right word. "extend" or "grow" maybe?

I'd change "provides" to "gives"; shorter and stronger. 

I think "API" should be plural. We have several that are at least very separated in the docs (ERTP, Zoe, Wallet...).